src/components/navigation/Navbar.js

```diff
@@ -192,15 +192,22 @@ export function Navbar(props) {
           </NavLink>
         </div>
         <div className="d-none d-md-block flex-grow-1" style={{ flexBasis: 0 }}>
-          <Widget
-            src="buildhub.near/widget/components.buttons.JoinNow"
-            config={{
-              redirectMap: redirectStore.redirectMap,
-            }}
-            props={{
-              children: <UserDropdown {...props} />,
-            }}
-          />
+          {props.signedIn ?
```

Comment:
Review the acceptance criteria again; this should show JoinNow if not logged in, then on click redirects to /join where you can login. "Join Now" is the call to action on gateway rather than to login

If logged in, then we show profile.

I think the change would be in [the JoinNow Button](https://github.com/NEARBuilders/gateway/blob/main/apps/builddao/widget/components/buttons/JoinNow.jsx)